mbslave/replication.py

```
@@ -238,44 +250,119 @@ def check_table_exists(db, schema, table):
     return True
 
 
+def download_url(url: str, dest_path: str) -> None:
+    headers = {}
+    file_mode = 'wb'
+    resume_size = 0
+    if os.path.exists(dest_path):
+        resume_size = os.path.getsize(dest_path)
+        headers['Range'] = 'bytes=%d-' % resume_size
```

Ruff (0.14.10)

Use format specifiers instead of percent format

(UP031)